CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.3.0 (2020-03-10)
+
+- Added the country field component. [#55](https://github.com/blackbaud/skyux-lookup/pull/55)
```

Comment:
Should "component" be "module"? I only ask because I realized we refer to "module" in the intro of the docs, so I want to make sure our terminology is OK in both places.

Reply:
Hmmm. I think component is right here. Maybe we should have used component on the docs. I'll touch base with you in Slack about if we want to change that during the demo work.

Reply:
Sounds good. I thought that might be the case. I'll make a note to update the docs.